src/app/config.rs

```diff
@@ -99,7 +99,7 @@ pub struct Config {
     pub checker_id: String,
```

Comment:
I wonder if it would actually make sense to move the number parsing into the deserialization (via a custom deserializer)

Since then you could produce an error message here in the config bit. Do we need the "full name" of the checker.

alternatively, why not jsut have the checker_id be just a number?